EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmploeeComponent.cs

```diff
@@ -1,35 +1,30 @@
-﻿using OpenQA.Selenium;
+﻿using EasyRestProjectNetTeam2.Decorator;
+using EasyRestProjectNetTeam2.EasyRestPages;
+using OpenQA.Selenium;
 using SeleniumExtras.PageObjects;
 
-namespace EasyRestProjectNetTeam2.EasyRestPages
+namespace EasyRestProjectNetTeam2.EasyRestComponentsObj
 {
-    public class AdministratorsManagePage : BasePage
+    public class AddEmploeeComponent : BasePage
```

Comment:
please rename to  AddEmplo**y(!!!)**eeComponent

Reply:
Done